<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtAdminPanel</class>
 <widget class="QWidget" name="QtAdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtAdminPanel</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Marka:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>siła silnika:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Nadwozie:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Spalanie:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Cena:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="carBrandLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="carModelLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="enginePowerSpinBox"/>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="carBodySpinBox"/>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="carPricespinBox"/>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="fuelConsumptionDoubleSpinBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>151</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="addPushButton">
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="modifyPushButton">
      <property name="text">
       <string>Modyfikuj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deletePushButton">
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
